### PR TITLE
docs: improved electron setup 

### PR DESCRIPTION
**Pull request by blu3beri**
- Improved the setup guide for electron
- Added some information to the macos part of the libindy setup 
  - M1 still needs some work...

### CODE REVIEW

File: docs/setup-electron.md [hunk 1]
```diff
@@ -2,84 +2,153 @@
...
-function wrapIndyCallWithErrorHandling(func: any) {
+const wrapIndyCallWithErrorHandling = (func: any) => {
   return async (...args: any[]) => {
     try {
       return await func(...args)
     } catch (e) {
-      e.name = 'IndyError'
-      e.indyName = e.message
-      throw e
+      if (e instanceof Error || e instanceof AriesFrameworkError || e instanceof IndySdkError) {
```
[the remark this answers is not in the document]
Comment (JamesKEbert) [reply docs/setup-electron.md:110]: Definitely interested in this alteration as well! I really need to take a look at the associated Toolbox PR still

File: docs/libindy/macos.md [hunk 2]
```diff
@@ -4,6 +4,16 @@
...
 
+> A better test would be to clone the [indy-sdk](https://github.com/hyperledger/indy-sdk/) and run:
+>
+> ```
+> cd indy-cli
+> cargo build --release
+> cd ./target/release/
+> ./indy-cli
+> wallet create 123 key
+> ```
```
Comment (JamesKEbert) [inline docs/libindy/macos.md:15]: Should we just replace the old method then?

**Review by JamesKEbert — COMMENTED** [review]
Great work @blu3beri :)

File: docs/libindy/macos.md (hunk 2 above)
Comment (blu3beri) [reply docs/libindy/macos.md:15]: This method would require cloning the indy-sdk repository and building via Cargo. The other check might be a bit less accurate, but it only requires the `ls` command. 

The issue with this test is that libindy can be installed correctly for the indy cli, but not for node. 

I will see if there is a better way, e.g. clone a small repo that just runs `indy.createWallet({id:'12'},{key:'abc'});` to test.

Comment (TimoGlastra) [reply docs/libindy/macos.md:15]: A thought. What if we add a binary to the `@aries-framework/node` package. This way you can just use `npx` to check if the installation was succeful

With [this PR](https://github.com/hyperledger/aries-framework-javascript/pull/510) merged we will be able to (@blu3beri I copied your example from https://github.com/animo/test-libindy-installation-for-node)

```
npx -p @aries-framework/node is-indy-installed

# output
# Libindy was installed correctly
```

What do you think?

Comment (blu3beri) [reply docs/libindy/macos.md:15]: That's awesome! Way better than what I had in mind.

**Review by TimoGlastra — APPROVED** [review]
LGTM @blu3beri. I'm going to merge this now as it already is a huge improvement on the current docs. We can then fix the last point with the indy error in a follow up PR :)

**Comment by JamesKEbert** [comment]
Good work here @blu3beri, thank you ❤️

**Pull request merged**